src/initialize_dae.jl

```diff
@@ -225,7 +227,7 @@ function _initialize_dae!(integrator, prob::DAEProblem,
     tmp = get_tmp_cache(integrator)[1]
     resid = get_tmp_cache(integrator)[2]
 
-    dt = t != 0 ? min(t / 1000, dtmax) : dtmax # Haven't implemented norm reduction
+    dt = t != 0 ? min(t / 1000, dtmax / 10) : dtmax / 10 # Haven't implemented norm reduction
```

Comment:
@ChrisRackauckas is there a reason this isn't checking for `alg.initdt`?

Reply:
maybe it should